_events/sr2019/southampton-kickstart.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Southampton Kickstart
+date: 2018-11-10 10:00:00
+layout: event
+type: kickstart
+location: Southampton University
+---
+
+Kickstart is the event which kicks off a competition year, where the game will
+be announced and robotics kits will be handed out to the teams.
+
+**Any team is welcome to attend the Southampton Kickstart, however you need to
```

Comment:
I'm not sure if this is entirely necessary? Seeing as the event is somewhat invite-only, and that the London event is _very_ invite-only, we can assume where people are going. We have MailChimp to tell us who's going where.

Reply:
ca1618b0556399947fdd64b6ec32c44e296f35a8